### ISSUE TITLE
ConvertTo-SecureString not working inside background jobs on Linux

### ISSUE THREAD

**Issue by janegilring** [PowerShell/PowerShell#13760]
Using ConvertTo-SecureString inside background jobs works on Windows, but fails on Linux. Tested both inside VS Codespaces and Azure Cloud Shell (repro below is frow the latter).

## Steps to reproduce

```powershell
Start-Job -Name test -ScriptBlock {ConvertTo-SecureString -AsPlainText -Force -String 'test'}
```

## Expected behavior

```none
System.Security.SecureString
```

## Actual behavior

```none
Id     Name            PSJobTypeName   State         HasMoreData     Location             Command
--     ----            -------------   -----         -----------     --------             -------
1      test            BackgroundJob   Failed        False           localhost            ConvertTo-SecureString -…
```

## Environment data

```none
Name                           Value
----                           -----
PSVersion                      7.0.3
PSEdition                      Core
GitCommitId                    7.0.3
OS                             Linux 4.15.0-1096-azure #106~16.04.1-Ubuntu SMP Thu Sep 10 18:51:54 UTC 2020
Platform                       Unix
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0…}
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1
WSManStackVersion              3.0
```

**Comment by vexx32** [comment]
What error is shown if you retrieve the results from the job?

**Comment by anmenaga** [comment]
On `Ubuntu 18` I see it failing on `PS 7.0.3`, but passing on `PS 7.1.0-rc.1`
@PaulHigin might have more info on this.

![PSJobError](https://user-images.githubusercontent.com/11860095/95506345-9e2c1200-0964-11eb-974f-1e20e82c4cf3.png)

**Comment by PaulHigin** [comment]
SecureString was not implemented on Linux platforms initially because of Windows only dependencies.  It was fixed in the 7.0 timeframe but only merged to 7.1 not 7.0.

https://github.com/PowerShell/PowerShell/pull/11185

**Comment by janegilring** [comment]
I see, thanks @PaulHigin 

@vexx32 No error or output at all
